raw_data/preprocessed/africa/senegal_french.txt

```diff
@@ -0,0 +1,259 @@
+
+(63) Sont interdits tous déversements, écoulements, rejets, dépôts directs ou indirects de toute nature susceptibles de provoquer ou d'accroître la pollution des eaux continentales et/ou eaux de mer dans les limites territoriales.
+
+(64) Sans préjudice des dispositions spéciales des conventions internationales portant prévention et répression de la pollution marine ratifiées par le Sénégal, sont interdits les déversements, les immersions et incinérations dans les eaux marines sous juridiction sénégalaise, de substances de toute nature susceptibles: [•] de porter atteinte à la santé publique et aux ressources marines biologiques; [•] de nuire aux activités maritimes, y compris la navigation et la pêche; [•] d'altérer la qualité des eaux marines du point de vue de leur utilisation; [•] de dégrader les valeurs d'agrément et le potentiel touristique de la mer et du littoral. Un arrêté interministériel précise, en tant que de besoin, la liste de ces substances.
+
+(65) Le Ministre chargé de l'environnement peut autoriser le déversement, l'immersion et l'incinération en mer de substances non visées dans la liste prévue en application de l'article 61 dans des conditions telles que ces opérations ne portent pas atteinte au milieu marin et à ses utilisations. Les Ministres chargés de l'environnement, de la marine marchande, de la pêche, de la santé et de l'intérieur fixent les conditions de mise en oeuvre de ces autorisations, qui doivent être, à la demande du Ministre chargé de l'environnement, précédées d'une étude d'impact démontrant leur innocuité.
+
+(66) Dans les cas d'avaries ou d'accidents survenus dans les eaux marines sous juridiction sénégalaise à tout navire, aéronef, engin ou plate-forme transportant ou ayant à son bord des hydrocarbures ou des substances nocives ou dangereuses et pouvant créer un danger grave et imminent pour le milieu marin et ses ressources, le propriétaire , l'exploitant ou le capitaine dudit navire, aéronef, engin ou plate-forme doit être mis en demeure par les autorités maritimes compétentes, en application du Code de la Marine marchande. Lorsque cette mise en demeure reste sans effet ou n'a pas produit les effets attendus dans le délai imparti, ou d'office en cas d'urgence, les autorités compétentes suscitées doivent faire exécuter les mesures nécessaires aux frais de l'armateur, de l'exploitant ou du propriétaire et en recouvrer le montant du coût auprès de ce dernier.
```

Comment:
The space between "le propriétaire" and the following comma should be removed.

Reply:
I will update.